server/configuration.go

```diff
@@ -162,6 +172,10 @@ func (c *configuration) Clone() *configuration {
 		}
 	}
 
+	if c.MaxCallParticipants != nil {
+		cfg.MaxCallParticipants = model.NewInt(*c.MaxCallParticipants)
```

Comment:
Honestly, I feel like Hamlet now :skull:  

Part of me wants to avoid relying on `model` as much as possible but it's so tempting to save a line by using the util...